insecure/corruptible/factory.go

```diff
@@ -32,18 +39,21 @@ type ConduitFactory struct {
 	cm                    *component.ComponentManager
 	logger                zerolog.Logger
 	codec                 network.Codec
-	myId                  flow.Identifier
+	me                    module.Local
```

Comment:
can this be renamed to something more clear? `mod` or `module`?

Reply:
`me` is the conventional name we use for the local node info